rollup.config.js

```diff
@@ -0,0 +1,42 @@
+import typescript from 'rollup-plugin-typescript2';
+import { terser } from "rollup-plugin-terser";
+import * as path from "path";
+import * as fs from "fs";
+
+const BASE_PATH = process.cwd();
+const getConfigFile = (name) => {
+  const filePath = path.resolve(BASE_PATH, name);
+  if (fs.existsSync(path.resolve(BASE_PATH, name))) {
+    return filePath;
+  }
+
+  return path.resolve(BASE_PATH, 'node_modules', '@stoplight', 'scripts', name);
+};
+
+let projectRollupConfig = null;
+
+try {
+  ({ default: projectRollupConfig } = require("esm")(module)(path.resolve(process.cwd(), 'rollup.config.js')));
```

Comment:
Can we do `import * as esm from "esm"` up above or do we need to do this require-foo for it to work?